CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+## 2.0.1 (YYYY-MM-DD)
+
+> [!NOTE]
```

Comment:
We already had this note for `2.0.0` should we keep it here as well?

Reply:
I included it in 2.0.0 even though it was also in 1.16.0. The reason was that I though people would maybe fast track from earlier releases to 2.0.0. But I assume that people would always read the changelog for the major bump, so don't think we need to included it any further.